code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -304,18 +295,20 @@
 								add_attack_logs(src, C, "[src] has grabbed [I] out of [C]'s hand with a tentacle")
 								on_hit(I) //grab the item as if you had hit it directly with the tentacle
 								return 1
-							else
-								to_chat(firer, "<span class='danger'>You can't seem to pry [I] out of [C]'s hands!</span>")
-								add_attack_logs(src, C, "[src] tried to grab [I] out of [C]'s hand with a tentacle, but failed")
-								return 0
-						else
-							to_chat(firer, "<span class='danger'>[C] has nothing in hand to disarm!</span>")
-							return 0
+							to_chat(firer, "<span class='danger'>You can't seem to pry [I] out of [C]'s hands!</span>")
+							add_attack_logs(src, C, "[src] tried to grab [I] out of [C]'s hand with a tentacle, but failed")
+						C.visible_message("<span class='danger'>[C] is knocked over by [src]!</span>","<span class='userdanger'>A tentacle hits you in the chest and knocks you over!</span>")
+						add_attack_logs(src, C, "[src] knocked over [C] with a tentacle")
+						C.KnockDown(2 SECONDS) //Not useless with antidrop.
+						return 1
 
 					if(INTENT_GRAB)
-						C.visible_message("<span class='danger'>[L] is grabbed by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and pulls you towards [H]!</span>")
-						add_attack_logs(H, C, "[H] grabbed [C] with a changeling tentacle")
-						C.throw_at(get_step_towards(H,C), 8, 2, callback=CALLBACK(H, TYPE_PROC_REF(/mob, tentacle_grab), C))
+						C.visible_message("<span class='danger'>[L] is entangledd by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and wraps around your legs!</span>")
+						add_attack_logs(H, C, "[H] imobilised [C] with a changeling tentacle")
+						if(!iscarbon(H))
+							return 1
```

Comment:
This is a bool return despite not using the defines elsewhere in proc, `TRUE` please

Reply:
Done